adapters/info.go

```diff
@@ -166,6 +166,7 @@ func (infos BidderInfos) SupportsWebMediaType(bidder openrtb_ext.BidderName, med
 type BidderInfo struct {
 	Maintainer   *MaintainerInfo   `yaml:"maintainer" json:"maintainer"`
 	Capabilities *CapabilitiesInfo `yaml:"capabilities" json:"capabilities"`
+	AliasOf      string            `json:"alias_of,omitempty"`
```

Comment:
most everyone uses camel-case... including this same endpoint (with `mediaType`). Let's keep it consistent and use `aliasOf`.

Reply:
Ahh, other parts of the code base we used snake case, but better to be more locally consistent.